app/controllers/articles_controller.rb

```diff
@@ -33,6 +31,10 @@ def update
 
   private
 
+  def set_article
+    @article = current_user.articles.find(params[:id])
+  end
+
```

Comment:
**suggestion (non-blocking):** Inline this method.

Sometimes a little bit of duplication is a good thing. It would be easy for a developer to miss the `before` action and get confused about where we set `@article`. If we inline this method, then our code is easier to read.